dotCMS/src/main/java/com/dotmarketing/cms/urlmap/filters/URLMapFilter.java

```diff
@@ -1,156 +0,0 @@
-/*
```

Comment:
As these changes are made for a minor release, shouldn't this class be deprecated instead of deleted ?

Reply:
I agree, is @wezell aware about this change?

Reply:
this is a filter so I don't know if this is needly

Reply:
@nollymar we are moving away from the idea of major and minor releases - I think it is ok to delete.